package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "canopy-webpack-config",
+  "version": "1.0.0",
+  "description": "Some defaults for webpack configs at canopy",
+  "main": "src/canopy-webpack-config.js",
```

Comment:
notice that this points to the uncompiled `src` directory. And that I don't have any babel or compilation of any sort here. That's because I think we can get away without it. The fanciest thing I'm using is template strings which has been supported by node for a long time.